frontend/src/renderer/components/Process/ImageBubble.svelte

```diff
@@ -75,12 +40,16 @@
     {:else}
       <ImageIcon size="1.5x" />
       {filename}
+      <button on:click={remove}><TrashIcon size="1.25x" /></button>
```

Comment:
Do you think you could remove the border around this?

Reply:
I have tried many methods and the border does not seem to want to leave